packages/embed/e2e/applitools-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { screenSizeDesktop, screenSizeMobile } from './cypress-utils'
+
+const eyesCheck = async (name: string, mobile: boolean) => {
+  const { width, height } = mobile ? screenSizeMobile : screenSizeDesktop
+
+  cy.eyesOpen({
+    appName: 'Embed - Cypress',
+    ignoreCaret: true,
+    browser: {
+      width,
+      height,
+      name: Cypress.browser.name,
+    },
+    batchName: 'Embed',
+  })
+  cy.eyesCheckWindow(name)
+  cy.eyesClose()
+}
+
+export const eyesCheckDesktop = (name: string) => eyesCheck(name, false)
+
+export const eyesCheckMobile = (name: string) => eyesCheck(name, true)
```

Comment:
Lets make this a bit more readable:

```
import { Viewport } from './cypress-utils.ts'

const eyesCheck = async (name: string, { width, height }: Viewport) => { ... }

export const eyesCheckDesktop = (name: string) => eyesCheck(name, screenSizeDesktop)

export const eyesCheckMobile = (name: string) => eyesCheck(name, screenSizeMobile)
```

WDYT?

Reply:
sounds good